Tests/CoreDataFeedStoreTests.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  CoreDataFeedStoreTests.swift
+//  Tests
+//
+//  Created by Shilpa Bansal on 23/02/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import XCTest
+@testable import FeedStoreChallenge
```

Comment:
`@testable` is not needed in this file - you can remove it.

```suggestion
import FeedStoreChallenge
```